codegen/scala-gen/src/test/scala/com/akkaserverless/codegen/scalasdk/SourceGeneratorSuite.scala

```diff
@@ -20,6 +20,7 @@ import com.lightbend.akkasls.codegen.TestData
 
 class SourceGeneratorSuite extends munit.FunSuite {
   import SourceGenerator._
+  import com.lightbend.akkasls.codegen.SourceGeneratorUtils._
 
   test("it can determine the main package") {
```

Comment:
Maybe this test should move to `codegenCore` as well?

Reply:
I agree

Reply:
I'll move it in separate PR to unlock other things on top of this